### ISSUE TITLE
Optimize images

### ISSUE THREAD

**Issue by Keno** [JuliaLang/www.julialang.org#1025]
It would be nice to automatically optimize images during pre-processing. In particular, on the blog, we always limit images to 690px wide, so we should probably pre-process images that have that resolution, to avoid having to transfer the large originals. For bonus points, process a few extra versions for common mobile resolutions. We can use the `srcset` `<img>` attribute to specify multiple versions of the image. However, we should be careful not to break the quality on high density (retina) displays.

**Comment by tlienart** [comment]
Maybe a pragmatic approach here is to have a set of guidelines for blog writers that would be kept in a `BLOG-CONTRIBS.md` and that would be reviewed in the PR process. 

Doing this automatically for all existing blog pages is likely to break a lot of stuff but I guess we could do a `du -sh` in `/_assets/` and hunt down the woolly mammoths.

**Comment by logankilpatrick** [comment]
Images should be auto optimizes by the the Img bot, just enabled this.

**Comment by StarTrooper08** [comment]
Yes , the imgbot optimises the images without affecting the images quality.
I too use it.
